testcases/kernel/controllers/cgroup/cgroup_regression_test.sh

```diff
@@ -182,9 +182,9 @@ test_3()
 	fi
 
 	# Run the test for 30 secs
-	mount -t cgroup -o cpu xxx cgroup/
+	mount -t cgroup -o cpu,cpuacct xxx cgroup/
 	if [ $? -ne 0 ]; then
-		tst_resm TFAIL "Failed to mount cpu subsys"
+		tst_resm TFAIL "Failed to mount cpu,cpuacct subsys"
 		failed=1
 		return
 	fi
```

Comment:
Ideally these test should be changed to reuse the already mounted cgroup system cgroup path (if available).